OrbitService/ProcessServiceImpl.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORIBIT_SERIVICE_PROCESS_SERVICE_IMPL_H_
+#define ORIBIT_SERIVICE_PROCESS_SERVICE_IMPL_H_
+
+#include <memory>
+#include <string>
+
+#include "ProcessUtils.h"
+#include "services.grpc.pb.h"
+
+class ProcessServiceImpl final : public ProcessService::Service {
+ public:
+  grpc::Status GetProcessList(grpc::ServerContext* context,
+                              const GetProcessListRequest* request,
```

Comment:
Is it standard for a protobuf type to be named "GetXYZ"?  Could we use just "ProcessListRequest" instead?  To me GetProcessListRequest is a function name more than a type name.  Same for GetProcessListReply. Also,  should we use Request/Response rather than Request/Reply, I think it's more common to see Response rather than Reply.

Reply:
I am not aware of any naming conventions for protobufs. The convention used here is for Request and Reply/Response messages use rpc method name as a prefix, it is get in this case but for a rpc call with name "UpdateHookedFunctionList" it is going to be named "UpdateHookedFunctionListRequest/Response". I think we should preserve Get here for consistency.

I renamed Reply to Response.

Reply:
Ok, that's fair.